docs/posts/roadmap-2024-H1/index.qmd

```diff
@@ -0,0 +1,219 @@
+---
+title: "Ibis project 2024 roadmap"
+author: "Cody Peterson"
+date: "2024-02-10"
+image: commits.png
+categories:
+    - blog
+    - roadmap
+    - community
+---
+
+## Overview
+
+Welcome to the first public roadmap for the Ibis project! If you aren't familiar
+with the background of Ibis or who supports it nowadays, we recommend reading
+[why Voltron Data supports Ibis](../why-voda-supports-ibis/index.qmd) before the
+roadmap below.
+
+## 2024 roadmap
+
+We have a [public roadmap as a GitHub
+project!](https://github.com/orgs/ibis-project/projects/5)
+
+![Ibis roadmap](roadmap.png)
+
+We are early in our use of this GitHub project, so please pardon any
+disorganization as we get it up and running efficiently. In general, we have:
+
+- **Roadmap view**: consisting of meta-issues in their respective repositories
+ for high-level objectives of the Ibis project
+- **Triage view**: consisting of new issues across Ibis project repositories
+ that need to be triaged
+- **Backlog view**: consisting of issues that have been triaged (assigned a
+ priority) and are on the backlog
+- **TODO view**: consisting of issues that are in progress or ready to be worked
+ on soon
+- **Label-specific views**: consisting of issues for specific labels, like
+ documentation or a large refactor
+
+Right now, [the team at Voltron Data](../why-voda-supports-ibis/index.qmd) sets
+the roadmap and priorities. Over time as more contributors and organizations
+join the project, we expect this process to diversify and become more
+community-driven.
+
+### Overall themes
+
+Our top five themes for 2024 include:
```

Comment:
Love these themes. We should include a concise sentence to convey the value to the audience for each. For example, why many Ibis backends is important? because user can **implement once and execute anywhere**

Reply:
added sentences...open to more suggestions, I don't love all of them but it's something